lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,89 @@
-# 
\ No newline at end of file
+# [Networking] Hardware
+*Understanding components of a common home router*
+
+![Wi-Fi Router](.rsrc/wifi-router.jpeg)
+
+# Objectives
+- Understand common components of a home router
+- Learn network specific terminology
+- Learn what components could be useful to solve connection requirements
+
+# Introduction
+A total of 5 billion people around the world use the internet today based on report from April 2022. Many people are familar with a device called a router that allows them to connect to the internet. Home routers exist in various different forms and in this lesson you will learn what they represent. 
+
+1. [Home Router Components](#home-router-components)
+2. [Switch](#switch)
+3. [Router](#router)
+4. [Broadband Connection](#broadband-connection)
+5. [Wireless Access Point](#wireless-access-point)
+6. [Firewall](#firewall)
+
+# Lesson
+## Home Router Components
+Home routers typically contain following components:
+* Switch (Ethernet LAN)
+* Router (Ethernet WAN)
+* Broadband Connection
+* Wireless Access Point (Wi-Fi)
+* Firewall 
+
+It is important to note that these are typical components, but a router can have more or less depending on the specific application of the device. 
+
+![image](https://user-images.githubusercontent.com/18094862/179405675-5a1c6c1b-4f70-4b8e-89a4-982c522729c4.png)
+
+## Switch
+Switches facilitate sharing of resources on a Local Area Network (LAN) including all of the computers, printers, and servers. All of the connected devices can share information and usually these are all of the devices you own and trust. Ethernet cables are used to connect devices on the network. 
+
+![image](https://user-images.githubusercontent.com/18094862/179407510-eecacf84-1513-4565-a9e4-55acc99d9fc0.png)
```

Comment:
include a picture of a switch.

label this ethernet cable

Reply:
Added